app/agent/rest/agent-version1.js

```diff
@@ -0,0 +1,10 @@
+angular.module('portainer.agent')
```

Comment:
What do you think about relocating these files to a `v1` folder? Would make it easier to track/remove them later on when we deprecate old agent versions support.